pipeline/tasks/protoc_tasks.py

```diff
@@ -325,6 +325,18 @@ def modify_imports(self, go_import_base, line):
 
 
 class GrpcPackmanTask(packman_tasks.PackmanTaskBase):
-    def execute(self, language, api_name, output_dir, packman_flags=[]):
-        arg_list = [language, api_name, '-o', output_dir] + packman_flags
-        self.run_packman(*arg_list)
+    def execute(self, language, api_name, output_dir, src_proto_path,
+                import_proto_path, packman_flags=[]):
+        api_name = task_utils.packman_api_name(api_name)
+        arg_list = [language, api_name, '-o', output_dir]
+
+        def remove_api_dirs(str):
+            return str.split(api_name)[0]
+        imps = map(remove_api_dirs, import_proto_path)
+        srcs = map(remove_api_dirs, src_proto_path)
+
+        arg_list += [arg for imp in imps for arg in ('-i', imp)]
+        for src in srcs:
+            args = arg_list[:]
+            args += ['-r', src] + packman_flags
```

Comment:
Note: per offline discussion, let's alter https://github.com/googleapis/googleapis/pull/55 so that packman can accept multiple `-r` flags. Then we don't have to worry about the overwriting.